### DISCUSSION TITLE
gatsby-source-wordpress ACF and Cloudinary

### DISCUSSION THREAD

**Discussion by BURROO** [gatsbyjs/gatsby#24202] [Help]
I am wondering what is the proper way to use wordpress headless and synchronize alle the media with cloudinary. I am using ACF Pro to setup the fields. Before installing the Cloudinary Plugin in wordpress I was able to retrieve the ACF image galleries in my graphql query. After installing it, all the ACF Gallery disappear in my graphql. After deactivating it, they appear again (but with the wordpress url). If I make request with postman to the Wordpress REST API I get the images as an array with all the information needed (and with the cloudinary url as expected), for some reason the creation for image nodes is failing in graphql.

**Comment by jasonbahl** [comment]
@BURROO based on the information provided, it's difficult to determine if there's anything that needs to change on the Gatsby side or the WordPress side. 

Based on the description that things weren't working when you activated the cloudinary plugin in WordPress, my intuition leads be to believe there might be a conflict between the Cloudinary plugin and the ACF Pro plugins, but it's hard to say for sure.

In order for us to be able to provide more assistance, can you provide a link to a repo with your Gatsby code, and provide details about your WordPress setup such as: 

- PHP Version
- WordPress Version
- Active Plugins (name and version)
- Active Theme (name and version)

With this information we can help determine if there's an issue with the way WordPress is behaving or if there's a bug or misconfiguration on the Gatsby side.

**Comment by BURROO** [comment]
@jasonbahl thank you for the quick response!
I hope this information helps:

PHP							Version 7.3
WORDPRESS					Version 5.4.1

ACTIVE PLUGINS
ACF Medium Editor Field		Version 2.5.3	 
ACF to REST API				Version 3.2.0
Advanced Custom Fields PRO	Version 5.8.11
Classic Editor					Version 1.5
Clean WP Admin Menu			Version 1.0.1
Cloudinary					Version 2.0.3
Compress JPEG & PNG images	Version 3.2.1 
Duplicate Post				Version 3.2.4
User Role Editor				Version 4.54
Webhook Netlify Deploy			Version 1.1.3

ACTIVE THEME				Twenty Twenty

**Comment by BURROO** [comment]
@jasonbahl  does the provided info help?
I am still struggling with this problem.
